community.html

```diff
@@ -1,34 +0,0 @@
----
-layout: page
-title: Community
-permalink: /community/
----
-<p>To chat with players and browse live games, download the game and join the lobby.</p>
-<p>Without the lobby, you can join the <a href="https://gitter.im/triplea-game/social">gitter discussion channel</a></p>
-<div class="community-col-wrapper">
-  <div class="community-col">
-    <a href="https://github.com/triplea-game/triplea">
-      <h2>Open Source on Github</h2>
-      <img src="../images/mark-github.svg" alt="Github Icon">
```

Comment:
I don't think those images are being used anywhere else in the webpage, so try removing them as well please. 
The travis build fails if an image that's referenced somewhere gets removed, so we're on the safe side.

Reply:
I could not find any usages either, removed in: 3d907cd